ios/FluentUI/Vnext/List/List.swift

```diff
@@ -153,6 +198,15 @@ extension MSFListView {
                 }
             })
             .buttonStyle(ListCellButtonStyle(tokens: tokens, layoutType: layoutType))
+            if let children = cell.children, isExpanded == true {
```

Comment:
If you use the isExpanded property from the MSFListVnextCellData, you probably don't need the @Sate isExpanded var in this View.